src/ImageSharp/IO/Endianness.cs

```diff
@@ -1,21 +0,0 @@
-﻿// Copyright (c) Six Labors and contributors.
-// Licensed under the Apache License, Version 2.0.
-
-namespace SixLabors.ImageSharp.IO
-{
-    /// <summary>
-    /// Endianness of a converter
-    /// </summary>
-    internal enum Endianness
```

Comment:
Debate on taste 3 .. 2 .. 1 ...

For me - fine. @JimBobSquarePants ?

Reply:
Fine by me, simplicity always wins.